cmd/nerdctl/run_restart.go

```diff
@@ -65,3 +65,10 @@ func updateContainerStoppedLabel(ctx context.Context, container containerd.Conta
 	})
 	return container.Update(ctx, containerd.UpdateContainerOpts(opt))
 }
+
+func updateContainerResatrtPolicyLabel(ctx context.Context, container containerd.Container, policyLabel string) error {
```

Comment:
We should check the capabilities of the restart plugin, https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/run_restart.go#L36-L50

Reply:
thanks